pkg/sdn/plugin/multitenant.go

```diff
@@ -111,6 +111,11 @@ func (mp *multiTenantPlugin) UpdateNetNamespace(netns *osapi.NetNamespace, oldNe
 }
 
 func (mp *multiTenantPlugin) DeleteNetNamespace(netns *osapi.NetNamespace) {
+	// Force multicast disabled for this VNID.  It hasn't been deleted from
+	// the VNIDMap until after this function returns, but UpdateLocalMulticastRules()
```

Comment:
("that behavior" = "VNIDMap doesn't get updated until after DeleteNetNamespace is called")